auto_py_to_exe/web/js/i18n.js

```diff
@@ -357,7 +369,8 @@ const translationMap = {
         ko: '출력 디렉토리',
         bg: 'Изходяща директория',
       },
-      increaseRecursionLimit: {
+      increaseRecursionLimhe: '',
```

Comment:
I see you checked "I have ran the application to make sure my code runs" - you may want to run it again as it appears this stops the application from loading.

Reply:
I have. must be something of last change I haven't check specifically. 
I've resolved now